src/components/TreeGrid/TreeGrid.tsx

```diff
@@ -266,30 +267,6 @@ export class TreeGrid extends React.PureComponent<ITreeGridProps, ITreeGridState
         );
     }
 
-    private formatBoolCell(cellData: any, rowData: any) {

```

Comment:
what happened here? leftover from a previous move?

Reply:
Yes, it's a sneaky delete, I forgot to clear this last time :)